test/junit/scala/tools/nsc/backend/jvm/opt/CompactLocalVariablesTest.scala

```diff
@@ -65,8 +62,8 @@ class CompactLocalVariablesTest extends ClearAfterClass {
     val varOpSlots = convertMethod(withCompact).instructions collect {
       case VarOp(_, v) => v
     }
-    assertTrue(varOpSlots.toString, varOpSlots == List(1, 2, 4, 5, 7, 8, 10, 11,  // stores
-                                                       1, 7, 2, 8, 4, 10, 5, 11)) // loads
+    assertTrue(varOpSlots == List(1, 2, 4, 5, 7, 8, 10, 11,  // stores
+                                                       1, 7, 2, 8, 4, 10, 5, 11), varOpSlots.toString) // loads
```

Comment:
let's try to keep the original bespoke formatting

```suggestion
    assertTrue(varOpSlots == List(1, 2, 4, 5, 7, 8, 10, 11,  // stores
                                  1, 7, 2, 8, 4, 10, 5, 11), // loads
               varOpSlots.toString)
```